_includes/schedule.html

```diff
@@ -2,7 +2,9 @@
 <section id="schedule" class="schedule">
     <div class="content-wrapper">
 
-        <h5>Check back for our schedule for CarpentryCon 2022!</h5>
+        <h5>We plan to post the schedule after June 17 ...</h5>
```

Comment:
I would recommend removing the "..." here.
Also, is this timeline correct if submissions close Sunday June 12, will we have a schedule on June 17?

Reply:
Good point. I was going by the outlined schedule from the program committee but will change that to be a bit less specific.